src/backend/executor/spi.c

```diff
@@ -1947,9 +1947,7 @@ _SPI_prepare_plan(const char *src, SPIPlanPtr plan)
 		 * Parameter datatypes are driven by parserSetup hook if provided,
 		 * otherwise we use the fixed parameter list.
 		 */
-		if (parsetree == NULL)
-			stmt_list = NIL;
-		else if (plan->parserSetup != NULL)
+		if (plan->parserSetup != NULL)
```

Comment:
At [line 2150](https://github.com/arenadata/gpdb/pull/732/files#diff-eca689b02874d585dbe0547b5b5af6e723a88247c89b662fa0f601f3d363d4f5R2150) there is the same parsetree pointer. If you remove check here, you should remove check there.

Reply:
removed too